tap_twilio/streams.py

```diff
@@ -12,313 +12,125 @@
                 'params': {},
-                'pagingation': 'none'
+                'pagination': 'root',
             },
-            # Reference: https://www.twilio.com/docs/usage/api/keys#read-a-key-resource
-            'keys': {
-                'api_url': 'https://api.twilio.com',
-                'api_version': '2010-04-01',
-                'path': 'Accounts/{ParentId}/Keys.json',
-                'data_key': 'keys',
+            'tasks': {
+                'api_url': 'https://taskrouter.twilio.com',
+                'api_version': 'v1',
+                'path': 'Workspaces/{ParentId}/Tasks',
+                'data_key': 'tasks',
                 'key_properties': ['sid'],
-                'replication_method': 'INCREMENTAL',  # Fetch ALL, filter results
+                'replication_method': 'FULL_TABLE',
                 'replication_keys': ['date_updated'],
                 'params': {},
-                'pagingation': 'root'
+                'pagination': 'root',
             },
-            # pylint: disable=line-too-long
-            # Reference: https://www.twilio.com/docs/sms/api/message-resource#read-multiple-message-resources
-            'calls': {
-                'api_url': 'https://api.twilio.com',
-                'api_version': '2010-04-01',
-                'path': 'Accounts/{ParentId}/Calls.json',
-                'data_key': 'calls',
+            'task_channels': {
+                'api_url': 'https://taskrouter.twilio.com',
+                'api_version': 'v1',
+                'path': 'Workspaces/{ParentId}/TaskChannels',
+                'data_key': 'task_channels',
                 'key_properties': ['sid'],
-                'replication_method': 'INCREMENTAL',  # Filter query
-                'replication_keys': ['end_time'],
-                'bookmark_query_field_from': 'EndTime>',  # Daily
-                'bookmark_query_field_to': 'EndTime<',
+                'replication_method': 'FULL_TABLE',
+                'replication_keys': ['date_updated'],
                 'params': {},
-                'pagingation': 'root'
+                'pagination': 'root',
             },
-            # pylint: disable=line-too-long
-            # Reference: https://www.twilio.com/docs/voice/api/conference-resource#read-multiple-conference-resources
-            'conferences': {
-                'api_url': 'https://api.twilio.com',
-                'api_version': '2010-04-01',
-                'path': 'Accounts/{ParentId}/Conferences.json',
-                'data_key': 'conferences',
+            'task_queues': {
+                'api_url': 'https://taskrouter.twilio.com',
+                'api_version': 'v1',
+                'path': 'Workspaces/{ParentId}/TaskQueues',
+                'data_key': 'task_queues',
                 'key_properties': ['sid'],
-                'replication_method': 'INCREMENTAL',  # Filter query
+                'replication_method': 'FULL_TABLE',
```

Comment:
Same as above. `FULL_TABLE` is not compatible with the `'replication_keys': ['date_updated'],`, and it should be `INCREMENTAL` if the API endpoint is supporting it.